packages/quantic/force-app/main/default/lwc/quanticPager/e2e/pagerObject.ts

```diff
@@ -0,0 +1,72 @@
+import {Page} from '@playwright/test';
+import {isUaCustomEvent} from '../../../../../../playwright/utils/requests';
+
+export class PagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get nextPageButton() {
+    return this.page.locator('button[title="Next Page"]');
+  }
+
+  get previousPageButton() {
+    return this.page.locator('button[title="Previous Page"]');
+  }
+
+  get pageButtons() {
+    return this.page.locator('c-quantic-number-button button');
+  }
+
+  pageButton(pageNumber: number) {
+    return this.pageButtons.nth(pageNumber - 1);
+  }
+
+  async goToLastPage() {
+    await this.pageButtons.nth(-1).click();
+  }
```

Comment:
I know it's implicit by the `async` but specifying the "type" of a promise also can be useful if you were to return it's value too.

For example, if `Locator.click()` were to return a value (it doesn't but it's just an example) we could receive it like this:

```suggestion
  async goToLastPage(): Promise<void> {
    return await this.pageButtons.nth(-1).click();
  }
```

Reply:
Very good point indeed, we should benefit from typescript fully!